package.json

```diff
@@ -65,34 +73,29 @@
     "yargs": "^15.4.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.5",
-    "@babel/plugin-transform-runtime": "^7.10.5",
-    "@babel/preset-env": "^7.10.4",
-    "@babel/runtime": "^7.10.5",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
```

Comment:
I think we can remove `@commitlint/cli` and `@commitlint/config-conventional` too.

Reply:
Done :+1: